make.go

```diff
@@ -495,7 +495,7 @@ func createGitRepository(debsrc, gopkg, orig string, u *upstream,
 		}
 		// Beginning newline in case the file already exists and lacks a newline
 		// (not all editors enforce a newline at the end of the file):
-		if _, err := f.Write([]byte("\n.pc\n")); err != nil {
+		if _, err := f.Write([]byte("\n/.pc/\n/_build/\n")); err != nil {
```

Comment:
`"\n/.pc/\n/_build/\n"` this looks rather strange no?
shouldn't it be `"\n.pc\n_build\n"` ?

Reply:
This tells git to ignore these if they are directories and are sitting at the root of the repository. Otherwise this will show them as untracked, as those would be unusual files to have, and conflict with the expectations of the tools.

Reply:
Didn't know that! Looks good to me then.